.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
This prevents people from opening a empty issues. All issues go through the forms after merging this PR.